Explorer/Assets/DCL/Communities/CommunitiesCard/CommunityCardController.cs

```diff
@@ -115,12 +123,42 @@ public override void Dispose()
             eventListController?.Dispose();
         }
 
+        private void OnDeleteCommunityRequested()
+        {
+            communityOperationsCancellationTokenSource = communityOperationsCancellationTokenSource.SafeRestart();
+            DeleteCommunityAsync(communityOperationsCancellationTokenSource.Token).Forget();
```

Comment:
Can't do this. CommunityCardController is disposable. If I dispose of it while this task is still running, it will try to use resources that have been disposed of. Your options:
- Change ControllerBase to be IAsyncDisposable and await this task in DisposeAsync.
- Solemnly swear that you have analysed every possible execution path of this task and have added code to handle every possible case of potentially accessing disposed resources.